charts/clearblade-iot-enterprise/charts/cb-redis/templates/redis-config.yaml

```diff
@@ -3,5 +3,7 @@ kind: ConfigMap
 metadata:
   name: redis-config
   namespace: {{ default "clearblade" .Values.global.namespace }}
+  labels:    
+{{ include "cb-redis.labels" . | indent 4 }}
 data:
-{{ (.Files.Glob "configs/pg_hba.conf").AsConfig | indent 2 }}
\ No newline at end of file
+{{ (tpl (.Files.Glob "configs/redis.conf").AsConfig .) | indent 2 }}
```

Comment:
this redis.conf file is showing as blank for me, did you just mean to rename the existing config?

Reply:
This is intentional, as our old redis conf was not actually being used, so it was sticking to defaults. However, I will add protected mode = off with the version bump.